CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+1. [Announcing](#announcing) the new notebooks to the community; and 
+1. [Logging any bugs](#logging-bugs) that are found in the Code Gallery.
+1. Managing the IOOS Data Demo Center [Project Board](#managing-ioos-data-demo-center-project-board).
+1. [Local management](#local-management-of-notebooks_demo) of the notebooks_demo repository.
+
+## Requesting new notebooks
+* New notebooks should be requested by adding a [ticket](https://github.com/ioos/ioos_code_lab/issues/new) to this GitHub repository. The ticket should contain the following information:
+```
+- [ ] What is language(s) for used in the example?
+- [ ] Is it focused on a particular module/software or an IOOS data service?
+- [ ] Can you provide a minimum example of the expected code and results in a notebook?
+
+Please provide a detailed description of the suggested example below:
+```
+
+### For the admin
+When the ticket gets created, make the following additions to the ticket:
+* Add the label [Notebook idea](https://github.com/ioos/ioos_code_lab/labels/Notebook%20idea).
+* Add the ticket to the [IOOS CodeLab](https://github.com/orgs/ioos/projects/1#card-49928448) project.
+* Put the card in the column [backlog](https://github.com/orgs/ioos/projects/1#column-5010196).
+
+## Building new notebooks
+If you would like to work on a notebook that has been proposed, follow these steps:
+1. [Create a fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of the [notebooks_demos](https://github.com/ioos/ioos_code_lab) repository.
+1. On your fork, [create a branch](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository) where you will start working on the new notebook. 
+   1. Give the branch a name that conveys what the new notebook is. For example, a notebook on converting data into Darwin Core could have a branch name of `data2dwc`. This makes it easier to know which branch is dedicated to which new notebook.
+1. On the new branch in your fork, start creating the new notebook. 
+   1. Notebooks are kept in the [jupyterbook/content/Code Gallery](https://github.com/ioos/ioos_code_lab/tree/master/jupyterbook/content/Code%20Gallery) directory of this repository.
+      1. There are three directories where notebooks can be placed:
+         1. `data_access_notebooks` - 
+         2. `data_analysis_and_visualization_notebooks` -
+         3. `data_management_notebooks` - 
+   2. Typically, it's best to copy an existing (working) notebook to a new file. If you are creating a python notebook, copy one of the python notebooks. If you are creating an R notebook, copy one of the R notebooks, etc
+   3. Name the notebook following the convention `DRAFT-[short name].ipynb`. Where, `[short name]` is an abbreviation for what the notebook will cover. Look at the existing notebook filenames to get a sense of the short names used. 
```

Comment:
Why `DRAFT`? Renaming the notebook on git later can be confusing for some folks.

Reply:
should we recommend `[date]-[short name].ipynb` ?

Reply:
I'm OK with that. What do you think? It helps to order the post by date.

Reply:
I put the date recommendation in the most recent commit. In the end I don't think this matters that much, it's just a matter of being able to easily find the new notebook.